cmd/juju/common/cloud.go

```diff
@@ -93,3 +93,27 @@ func BuiltInClouds() (map[string]jujucloud.Cloud, error) {
 	}
 	return allClouds, nil
 }
+
+// AnyCloudByName returns a cloud for given name
+// regardless of whether it's public, private or builtin cloud.
+// Since this method caters for builtin clouds, unlike cloud.CloudByName,
+// it should be used in cmd.
+func AnyCloudByName(cloudName string) (*jujucloud.Cloud, error) {
```

Comment:
can this just be called "CloudByName"? the "Any" prefix doesn't explain the difference between this and cloud.CloudByName, so may as well not be there